Rakefile

```diff
@@ -39,6 +39,38 @@ task :rubygems_admins do
   end
 end
 
+task :update_dependencies do
+  puts "Updating all internal fastlane dependencies"
+
+  regex = %r{spec.add_dependency .(.+).\, .\>\= (\d+\.\d+\.\d+).\, .\< \d+\.\d+\.\d+.}
```

Comment:
I think it is OK, maybe just document the assumption in a comment for the code :+1: